### ISSUE TITLE
node v0.12.0 Connection Error

### ISSUE THREAD

**Issue by atomstrom** [chuckmo/grunt-ssh#102]
After updating to node v0.12.0 I get the following error when I try to log on a server using a private key for authentication:

```sh
Warning: Connection :: error :: Error: Disconnected by host (PROTOCOL_ERROR): Packet integrity error. Use --force to continue.
```

Any suggestions? Thanks!

**Comment by webdesignberlin** [comment]
Same Problem here on Windows and Mac. 

I tested an update of ssh2. but it did not work: After Updating the included ssh2-package from 0.3.0 to 0.4.4 i became a nother Error: [stream.push() after EOF](https://github.com/mscdex/ssh2/issues/233).

**Comment by bmatzner** [comment]
Updating the dependency of ssh2 to ~0.4.6 works fine. See #106
